packages/front-end/components/Settings/ApiKeysModal.tsx

```diff
@@ -55,6 +58,23 @@ const ApiKeysModal: FC<{
         })}
         {...form.register("environment")}
       />
+      <div className="mb-3 d-flex flex-column-reverse">
+        <Toggle
+          id={"encryptFeatures"}
+          value={!!form.watch("encryptFeatures")}
+          setValue={(value) => {
+            form.setValue("encryptFeatures", value);
+          }}
+        />
+        <span>
+          <label htmlFor="encryptFeatures">Encrypt Features?</label>
+          <Tooltip
+            className="pl-1"
+            body="Enabling this will encrypt the list of features returned from the GrowthBook API. "
+            tipMinWidth="200px"
+          />
+        </span>
+      </div>
```

Comment:
I think a lot of people will click this toggle, just because "encryption" = "safer", without realizing all of the other stuff they need to do to make it work.  How do we add this in such a way that doesn't frustrate users who just want something simple to start with?

Reply:
That's a fair call.

I've made a few tweaks based on this feedback.

The first, and completely optional, is I switched the fields around so the environment drop-down field is field, and the optional description field is second. It felt odd to have an optional field be the first field, and a required field be the second. This was exacerbated when the encrypt field was introduced as it then meant the one required field was then sandwiched by two optional fields.

The second change is introducing a pattern I saw on the metric form where we have a concept for "advanced fields". It's kind of a weird pattern since their is only a single advanced field, but it would accomplish the goal of not adding noise to the majority of organizations.

<img width="537" alt="Screen Shot 2022-10-06 at 10 06 30 AM" src="https://user-images.githubusercontent.com/75274610/194335246-3d9a96f7-8d37-458a-aeff-e11943e9e963.png">

And... this is the view when someone opts to show advanced fields
<img width="529" alt="Screen Shot 2022-10-06 at 10 06 39 AM" src="https://user-images.githubusercontent.com/75274610/194335434-add68b9a-0f6b-463c-81a8-1bb1472751f4.png">

And finally, I updated the language of the title and the tooltip to be a bit more contextual about exactly what this means. 
<img width="574" alt="Screen Shot 2022-10-06 at 10 10 53 AM" src="https://user-images.githubusercontent.com/75274610/194336938-2e85ed81-a832-417c-a957-b0094a19e54c.png">

Open to thoughts and different treatments.